govcd/vapp.go

```diff
@@ -802,16 +825,13 @@ func (vapp *VApp) DeleteMetadata(key string) (Task, error) {
 	return *task, nil
 }
 
-func (vapp *VApp) AddMetadata(key, value string) (Task, error) {
-	err := vapp.Refresh()
-	if err != nil {
-		return Task{}, fmt.Errorf("error refreshing vApp before running customization: %v", err)
-	}
-
-	if vapp.VApp.Children == nil {
-		return Task{}, fmt.Errorf("vApp doesn't contain any children, aborting customization")
-	}
+func (vapp *VApp) AddMetadata(key string, value string) (Task, error) {
```

Comment:
Please add an SDK-user targeted comment (documentation) about this public function.

Reply:
Fixed.